Project.toml

```diff
@@ -4,25 +4,20 @@ authors = ["Phillip Alday", "Douglas Bates", "Lisa DeBruine", "Reinhold Kliegl"]
 version = "0.2.0"
 
 [deps]
-DataFrames = "a93c6f00-e57d-5684-b7b6-d8193f3e46c0"
 Documenter = "e30172f5-a6a5-5a46-863b-614d45cd2de4"
+LinearAlgebra = "37e2e46d-f89d-539d-b4ee-838fcccc9c8e"
 MixedModels = "ff71e718-51f3-5ec2-a782-8ffcbfa3c316"
 PooledArrays = "2dfb63ee-cc39-5dd5-95bd-886bf059d720"
-ProgressMeter = "92933f4c-e287-5a05-a399-4b506db050ca"
 Random = "9a3f8284-a2c9-5f02-9a11-845980a1fd5c"
-StaticArrays = "90137ffa-7385-5640-81b9-e52037218182"
 Statistics = "10745b16-79ce-11e8-11f9-7d13ad32a3b2"
 Tables = "bd369af6-aec1-5ad0-b16a-f7cc5008161c"
 
```

Comment:
Would it be worthwhile adding PrettyTables?  It is comparatively lightweight and works for any `Tables.jl` type.

Reply:
Maybe. We're not actually using any custom types, so there's nothing to dispatch on. We could add that in the worked example and show users how they can use that. Or maybe just re-export `pretty_table` so that they can call that without explicitly loading another package?

Reply:
Yes, I think that re-exporting `pretty_table` would be the way to go.